.github/actions/list-variants/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "list-variants",
+  "version": "1.0.0",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "prepare": "ncc build index.js -o dist --source-map --license licenses.txt"
+  },
+  "dependencies": {
```

Comment:
Just FYI, `toml` is the only dependency we're using that isn't being used by the [other actions](https://github.com/actions/checkout/blob/main/package.json#L30) we have been using,